README.md

```diff
@@ -51,16 +48,15 @@ WANT TO CONTRIBUTE?
 
 Please, read the [contributing guidelines](.github/CONTRIBUTING.md) on how to create [Issues](.github/CONTRIBUTING.md#issues) and [Pull Requests](.github/CONTRIBUTING.md#pull-requests).
 
-**Note:** The team has changed, so there will be many changes in the upcoming versions.
-If you need informations about versions, check the [changelog](CHANGELOG.md).
+**Note:** The team has changed, so there will be many changes in the upcoming versions. If you need version information, check the [changelog](CHANGELOG.md).
```

Comment:
This reminds me that I need to update the changelog after releasing 0.6.0. I forgot about it :S